CHANGELOG.md

```diff
@@ -13,15 +13,16 @@ This Splunk OpenTelemetry Collector release includes changes from the [opentelem
 
 ### 🛑 Breaking changes 🛑
 
+- (Core) `expandconverter`: By default expandconverter will now error if it is about to expand `$FOO` syntax. Update configuration to use `${env:FOO}` instead or disable the `confmap.unifyEnvVarExpansion` feature gate. ([#10435](https://github.com/open-telemetry/opentelemetry-collector/pull/10435))
+  - A bug was identified where bash sysntax code that used expansion didn't honor escaping ([#10560](https://github.com/open-telemetry/opentelemetry-collector/pull/10560))
```

Comment:
```suggestion
  - A bug was identified where bash syntax code that used expansion didn't honor escaping ([#10560](https://github.com/open-telemetry/opentelemetry-collector/pull/10560))
```
I think this message should go up above the breaking changes with the other warnings, maybe even the first warning in the list. We should make it clear that the bug is in `v0.104.0`, not fixed in. I'd also suggest we include the exact case that's causing issues in the changelog, so customers know as clearly as possible whether or not they're impacted.